simulator.py

```diff
@@ -412,6 +429,11 @@ def at(self, location, duration):
                                 # latent={"infected":self.is_exposed}
                                 )
 
+        if location.location_type == 'hospital':
+            if random.random() > location.recovery_proba():
+                self.death = True
```

Comment:
death should be dependent on sickness and hospital's condition

Reply:
fixed. defined death as if there's no ventilators available in hospital and human is really sick.